templates/src/csharp/Grpc.Auth/project.json.template

```diff
@@ -34,9 +34,9 @@
           "net45"
         ],
         "dependencies": {
-          "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc2-24027",
```

Comment:
The rc2-24027 are prerelease versions, it makes sense to update to their GA versions (no particular reason, I just think it makes sense to update to stable dependencies once they are available).